lib/responder.js

```diff
@@ -1119,6 +1119,11 @@ function resolvedPromise(operation, resolve) {
   }
 
   var data    = operation.data;
+  for(let i=0; i<data.length; i++){
+    if(data[i].format === 'text' && typeof data[i].value !== 'string'){
+      data[i].value = data[i].value.toString();
+    }
+  }
   var hasData = (data != null);
```

Comment:
Given this null check here, I think you need to add checks before your code to make sure data is not null and that data has `length` as well.